src/main/java/nl/knaw/huc/resussun/timbuctoo/CollectionsMetadataMapper.java

```diff
@@ -11,33 +11,76 @@
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
-public class CollectionsMetadataMapper implements TimbuctooResponseMapper<Map<String, List<CollectionMetadata>>> {
+public class CollectionsMetadataMapper implements TimbuctooResponseMapper<Map<String, CollectionMetadata>> {
   private static final ObjectMapper MAPPER = new ObjectMapper();
+  private static final String COLLECTION_ID_PROP = "collectionId";
+  private static final String COLLECTION_LIST_ID_PROP = "collectionListId";
+  private static final String URI_PROP = "uri";
+  private final String key;
+
+  private CollectionsMetadataMapper(String key) {
+    this.key = key;
+  }
+
+  private static CollectionMetadata mapCollectionMetadata(JsonNode collection) {
+    String collectionId = collection.get(COLLECTION_ID_PROP).asText();
+    String collectionListId = collection.get(COLLECTION_LIST_ID_PROP).asText();
```

Comment:
There is no such property as `collectionListId` and therefore it fails. As it is not necessary anyway, we can just get rid of this line.